README.md

```diff
@@ -85,7 +85,7 @@ Contributions are welcome! Just submit a Pull Request or Github Issue.
 
 ```bash
 docker build -t dag-factory .
-docker run -d -p "8080:8080" -v "$(pwd):/app" dag-factory
+docker run -d -p "8080:8080" -v "$(pwd):/app" --network main_network dag-factory
```

Comment:
I would add a comment here, explaining that Airbyte needs to be ran in the same docker network